docgen/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+var join = require('path').join;
+
+module.exports = {
+  "extends": join(__dirname, "../.eslintrc.js"),
+  "settings": {
+    "import/resolver": {
```

Comment:
This lint module is very helpful, it already warned us when:
- the module/file you want to import does not exists
- the module is not present in package.json

Here we configure it to use the webpack resolver so that the webpack alias in the docs for "react-instantsearch" points to the local code